include/z64actor.h

```diff
@@ -493,6 +493,29 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+#define ACTOR_PARAMS_TRANSITION_INDEX_SHIFT 10
+#define ACTOR_TRANSITION_INDEX(actor) ((u16)(actor)->params >> ACTOR_PARAMS_TRANSITION_INDEX_SHIFT)
```

Comment:
I would name this DOOR_ instead of ACTOR, since it is expecting a specific actor (or type of actor) to access its params

Reply:
i think ideally it would live in a header specifically for the door actors, but know we probably wont do that yet cause of header discussions

Reply:
yeah the full name for `ACTOR_TRANSITION_INDEX` should be `TRANSITION_ACTOR_TRANSITION_INDEX`, I shortened it to `ACTOR_TRANSITION_INDEX`

I'm not a fan of `DOOR_` since that name doesn't include transition planes
Do you think `TRANSITION_ACTOR_INDEX` would be better maybe? `TRANSITION_ACTOR_TRANS_INDEX`? `TRANS_ACTOR_TRANS_INDEX`?

Reply:
`TRANSITION_ACTOR_INDEX` is probably ok, but yea i can see how naming this is pretty dumb

Reply:
Changed to `TRANSITION_ACTOR_INDEX`